### PR TITLE
Feature sc 64229 llm sdk support audio id for openai

### PR DESCRIPTION
**Pull request by hoangvvo**
## What does this PR do?

- Use json-schema instead of openapi schema
- support audio id for openai

## Type of change

This pull request is a
- [ ] Feature
- [ ] Bugfix
- [ ] Enhancement

Which introduces
- [ ] Breaking changes
- [ ] Non-breaking changes

## How should this be manually tested?

Check test case. Or test with `javascript/examples/generate-audio-multiturn.ts`

## Any background context you want to provide beyond Shortcut?

Lots of changes due to some refactoring (mainly tooling change of openapi -> json schema). only relevant commit for audio id support: https://github.com/firefliesai/llm-sdk/pull/5/commits/97e7780e7b4b21a0514a7696bf4bb27b627f5306

## Screenshots (if appropriate)

image

## Loom Video (if appropriate)

https://github.com/user-attachments/assets/82df9fb4-7b52-43ba-b623-3c297e63c5b2

## Any Security implications

xxx

## Release Notes

- **New Features**
	- Introduced a new JSON schema (`schema.json`) for the language model SDK, defining various message parts and their properties.
	- Added functionality for generating audio responses in a multi-turn conversation format.
	- Implemented a script for generating TypeScript definitions from the new JSON schema.

- **Bug Fixes**
	- Enhanced error handling across various modules with specific error classes for better clarity.

- **Documentation**
	- Updated references in the `README.md` to reflect the new schema file location.

- **Chores**
	- Updated dependencies and scripts in `package.json` for improved functionality and performance.

### CODE REVIEW

File: javascript/examples/generate-audio-multiturn.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        {
+          type: "text",
+          text: "Is a golden retriever a good family dog?",
+        },
+      ],
+    },
+  ],
+});
+
+console.dir(response, { depth: null });
```
Comment (FoysalOsmany) [inline javascript/examples/generate-audio-multiturn.ts:27]: We can remove the debug log

File: javascript/examples/generate-audio-multiturn.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        {
+          type: "text",
+          text: "What about a labrador?",
+        },
+      ],
+    },
+  ],
+});
+
+console.dir(response2, { depth: null });
```
Comment (FoysalOsmany) [inline javascript/examples/generate-audio-multiturn.ts:63]: debug log can be removed

**Review by FoysalOsmany — APPROVED** [review]
LGTM

File: javascript/examples/generate-audio-multiturn.ts (hunk 1 above)
Comment (hoangvvo) [reply javascript/examples/generate-audio-multiturn.ts:27]: This is just example code. We can keep it.

**Pull request merged**